src/main/java/com/epam/ta/reportportal/core/integration/plugin/impl/CreatePluginHandlerImpl.java

```diff
@@ -253,6 +230,15 @@ private IntegrationType startUpPlugin(String newPluginId, Optional<PluginWrapper
 
 			integrationType.setName(newLoadedPluginId.get());
 
+			Optional<ReportPortalExtensionPoint> instance = pluginBox.getInstance(newLoadedPluginId.get(),
+					ReportPortalExtensionPoint.class
+			);
+
+			if (instance.isPresent()) {
+				IntegrationDetailsProperties.COMMANDS.setValue(integrationType.getDetails(), instance.get().getCommandNames());
+				integrationType.setIntegrationGroup(IntegrationGroupEnum.valueOf(instance.get().getIntegrationGroup().name()));
```

Comment:
is it really necessary to get Integration group name and then create a enum from it?

Reply:
different enums. on is in dao, another is in plugin api